web-console/knikubevirt/link-between-PVC-VMdisk/link-between-PVC-VMdisk.md

```diff
@@ -4,12 +4,12 @@ In some cases, a disk is being backed by a Kubernetes Persistent Volume Claim (P
 In the OpenShift Console UI, we list these PVCs out for the user to view and edit.
 In our Virtualization UI, we should consider how we might show that a Disk is backed by a PVC and then link over to that PVC.
 
-Also, we should consider if we want to link the other way: if the user happens to be looking at the PVCs, we would want to link to a VM making use of a PVC.
+Also, we thought to link the other way: if the user happens to be looking at the PVCs, we would want to link to a VM making use of a PVC.
 
 ## Linking from PVC to Disk
 
 From the PVC page, we add a link to the table only if CNV is installed (or maybe even only if there are VMs) that will link to the VM. 
```

Comment:
Let's remove the reference to CNV and either talk about KubeVirt instead or just refer to VMs in general here.